runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/util/SchemaCoderCloudObjectTranslator.java

```diff
@@ -99,6 +99,9 @@ public SchemaCoder fromCloudObject(CloudObject cloudObject) {
       SchemaApi.Schema.Builder schemaBuilder = SchemaApi.Schema.newBuilder();
       JsonFormat.parser().merge(Structs.getString(cloudObject, SCHEMA), schemaBuilder);
       Schema schema = SchemaTranslation.schemaFromProto(schemaBuilder.build());
+      if (schema.isEncodingPositionsOverridden()) {
+        SchemaCoder.overrideEncodingPositions(schema.getUUID(), schema.getEncodingPositions());
+      }
```

Comment:
These calls aren't passing the null checker since `schema.getUUID()` returns a nullable UUID